edx_sga/static/js/src/edx_sga.js

```diff
@@ -16,6 +16,7 @@ function StaffGradedAssignmentXBlock(runtime, element) {
         var removeGradeUrl = runtime.handlerUrl(element, 'remove_grade');
         var template = _.template($(element).find("#sga-tmpl").text());
         var gradingTemplate;
+        var studentAssignments;
```

Comment:
Just making sure I'm following this.. We're storing this list of studentAssignments because we get partial updates back when a user updates their grades and therefore can't call `renderStaffGrading`?